src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/wwwroot/css/site.css

```diff
@@ -1,37 +1,23 @@
-﻿/* Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification\ 
```

Comment:
We don't want to link to this anymore?

Reply:
I'll add the link back, but this also points out that we have site.min.css (and .js) files checked in here which are out of date and we currently don't have a way to update them.

Reply:
Maybe ask @DamianEdwards what to do here because bundling & minification is always on his mind.

Reply:
Yeah, I contacted @danroth27 through other channels but @DamianEdwards would also be a help here.

Reply:
Talked to @danroth27 in person, he says we want to keep the minified files, but should just re-generate them when we make a change to content files.